resources/js/main.js

```diff
@@ -46,39 +46,33 @@ var JsRocks = function() {
 	* DEFINE
 	*
 	**/
-	var	HARMONIC = new Harmonic(),
-		W = window,
-		D = document,
-		PROTOCOL = W.location.protocol,
-		HOST = W.location.host,
-		HOSTNAME = W.location.hostname,
-		PATHNAME = W.location.pathname,
-		ORIGIN = W.location.origin,
-		INFORMATIONS = {},
-		TEMPLATE = {},
-		JSROCKS = {},
-		PRIVATE = {},
-		PUBLIC = this,
-		pageElements = {};
-
-
-	/**
+    var PROTOCOL      = window.location.protocol;
+    var HOST          = window.location.host;
+    var HOSTNAME      = window.location.hostname;
+    var PATHNAME      = window.location.pathname;
+    var ORIGIN        = window.location.origin;
+    var HARMONIC      = new Harmonic();
+    var INFORMATIONS  = {};
+    var TEMPLATE      = {};
+    var JSROCKS       = {};
+    var PAGEELEMENTS  = {};
```

Comment:
You can use underscores in constant names to improve readability: `PAGE_ELEMENTS`